buildkit/hypopg.yaml

```diff
@@ -7,8 +7,9 @@ source: https://github.com/HypoPG/hypopg/archive/refs/tags/1.4.0.tar.gz
 description: Hypothetical Indexes for PostgreSQL
 licence: PostgreSQL
 keywords:
-  - hypothetical indexes
+  - hypothetical
```

Comment:
`hypothetical indexes`  make more sense to me than just `hypothetical` as the keyword?

Reply:
@swijckmans In this case would it be suitable to have a keyword longer than 15 chars?

Reply:
Yes, I was not saying that as a hard limit but try using a single word as much as possible.

Reply:
Got it - will modify accordingly.